modules/tinymce/src/core/main/ts/api/WindowManager.ts

```diff
@@ -142,15 +142,15 @@ const WindowManager = (editor: Editor): WindowManager => {
      * Opens a new window.
      *
      * @method open
-     * @param {Object} args For a list of options, see: <a href="https://www.tiny.cloud/docs/ui-components/dialog/#configurationoptions">Dialog - Configuration options</a>.
+     * @param {Object} args For a list of options, see: <a href="https://www.tiny.cloud/docs/tinymce/6/dialog-configuration.html#_configuration_options">Dialog - Configuration options</a>.
```

Comment:
Another ID that should be fixed (i.e. not using an underscore prefixed ID)

- [ ] Fix this here and in the docs to use a fixed ID like it was previously